src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -186,12 +188,19 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     enabled: Boolean(accountId && wallet && accountMetadata),
   })
 
+  const memo = useMemo(() => {
+    if (isRunePool) return RUNEPOOL_DEPOSIT_MEMO
```

Comment:
Constructing in place is absolutely fine since we don't get a quote 🤝 although can we add this in `useSendThorTx` with a new `depositRunePool` in the `Type` Union?

Won't be an issue at runtime for deposit and is only for the sake of completeness, however for withdraws, things may be different since there is logic related to savers withdraw action:

https://github.com/shapeshift/web/blob/d11393b2e5a10eacf84876bdf4b5c7ccf2a24665/src/lib/utils/thorchain/hooks/useSendThorTx.tsx#L91-L93

Reply:
I'm not very aligned with this, as previously said, `useSendThorTx` is responsible of sending the TX, but not constructing the memo, and the action wouldn't be used

Reply:
Added the `action` to union in [19db6af](https://github.com/shapeshift/web/pull/7440/commits/19db6afed2d77fec1f3cb46386d3487d42dbb41e)